app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const app = express();
+
+const {
+  getTopics,
+} = require('./controllers/topics.controllers');
+
```

Comment:
We technically don't need this line until we handle a post. Try to avoid bringing in boilerplate code which isn't needed yet